src/views/Primary/index.tsx

```diff
@@ -44,6 +44,7 @@ const Primary: React.FC<PrimaryProps> = ({ contacts, companies }) => {
 		<div>
 			<div className="primary-header">
 				<span className="primary-header-text">Items</span>
+				<Button variant="tertiary" label="+ Create contact" onClick={() => history.push('/create')} />
```

Comment:
I think it'll be better to wrap it into the `<Link>` component of `react-router-dom` instead of passing `onClick`